### PR TITLE
Feature-91 Add linting tests tasks

### PR DESCRIPTION
**Pull request by rodrigo-baldiviezo-jalafund**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -31,6 +32,12 @@ steps:
     projects: $(projects)
     arguments: '--output $(Build.ArtifactStagingDirectory)'
 
+- task: DotNetCoreCLI@2
```
Comment (almartinch) [inline azure-pipelines.yml:31]: move entire task right before `DotNet Build` tasks please

File: azure-pipelines.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ pr:
   
 variables:
   projects: '**/*.sln'
+  testsProjects: '**/*Tests/*.csproj'
```
Comment (almartinch) [inline azure-pipelines.yml]: add this variable for me please: `buildConfiguration: $(buildConfiguration)`

we're gonna test using env variables for the pipeline :)

File: azure-pipelines.yml [hunk 3]
```diff
@@ -31,6 +32,12 @@ steps:
     projects: $(projects)
     arguments: '--output $(Build.ArtifactStagingDirectory)'
 
+- task: DotNetCoreCLI@2
+  displayName: 'DotNet Tests'
+  inputs:
+    command: test
+    projects:  $(testsProjects)
```
Comment (almartinch) [inline azure-pipelines.yml:35]: add the `arguments` input right after this line, something like this:
```yml
    arguments: '-c $(buildConfiguration)' 
```

File: azure-pipelines.yml [hunk 4]
```diff
@@ -31,6 +32,12 @@ steps:
     projects: $(projects)
     arguments: '--output $(Build.ArtifactStagingDirectory)'
```
Comment (almartinch) [inline azure-pipelines.yml]: include the build configuration argument into this call as well, please

File: azure-pipelines.yml (hunk 1 above)
Comment (almartinch) [reply azure-pipelines.yml:31]: no, nevermind... left it where it was

**Review by almartinch — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml [hunk 5]
```diff
@@ -13,6 +13,8 @@ pr:
   
 variables:
   projects: '**/*.sln'
+  testsProjects: '**/*Tests/*.csproj'
+  buildConfiguration: $(buildConfiguration)
```
Comment (martin-callejas-jalafund) [inline azure-pipelines.yml]: remove the whole variables sections, but keep the values for the first two, we're gonna add them through Azure interface https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch

**Review by martin-callejas-jalafund — APPROVED** [review]

**Pull request merged**